app/api/metrics.py

```diff
@@ -51,3 +63,61 @@ def getScoreCards(
     teamId = decodeJWT(token)["teamId"]
 
     return microserviceScoreCardService.getByTeamId(teamId)
+"""
+
+# ADD NEW METRIC HERE
+@router.post("/", response_model=schemas.Metric)
+def createMetric(metric: schemas.MetricCreate, metricCrud: crud.CRUDMetric = Depends(dependencies.getMetricsCrud)) -> schemas.Metric:
+    metricObj = metric
+    # change ' ' with '-'
+    metricObj.code = format_code(metric.name)
+    # Stringfying the list of areas to be saved as string 
+    metricObj.area = json.dumps(metric.area)
```

Comment:
check if area is not None before calling json.dumps
in general, json.dumps will throw exception in case something went wrong, make sure to handle this exception and throw an ExceptionCustom with the right status_code and a descriptive error message.